libs/api/rpc_client.h

```diff
@@ -13,7 +13,13 @@ RPC_STATUS
 clean_up_rpc_binding(void);
 
 ebpf_result_t
-ebpf_rpc_verify_program(ebpf_program_verify_info* info, const char** logs, uint32_t* logs_size) noexcept;
+ebpf_rpc_verify_program(
+    _In_ ebpf_program_verify_info* info,
```

Comment:
can this be const?   Often `_In_` implies it should be const, and non-const implies it should be `_Inout_` like in 19.
(There are valid exceptions like usually `_In_` is used with objects that have a lock in them that is taken inside.)

Reply:
I can change it to const, but then we need to immediately remove the const-ness before passing this to the RPC function "ebpf_client_verify_program".

Reply:
Ok, don't worry about it.